sessiontxn/isolation/readcommitted_test.go

```diff
@@ -376,6 +377,111 @@ func TestTidbSnapshotVarInRC(t *testing.T) {
 	tk.MustExec("rollback")
 }
 
+func TestErrorInInsertInRC(t *testing.T) {
```

Comment:
Can we move `TestErrorIn***InRC` to one method? Seems all of them have some common codes.

Reply:
Done